docs/howto/writing-an-application-from-scratch/src/main/resources/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example</groupId>
+  <artifactId>restful-endpoint</artifactId>
+  <name>Wildfly Swarm Example</name>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>war</packaging>
```

Comment:
By convention (https://maven.apache.org/developers/conventions/code.html), this block should look more like

```xml
<modelVersion>4.0.0</modelVersion>

<groupId>com.example</groupId>
<artifactId>restful-endpoint</artifactId>
<version>1.0.0-SNAPSHOT</version>
<packaging>war</packaging>

<name>Wildfly Swarm Example</name>
```

Reply:
Thanks, will do.